src/components/entry-boxes/entry-boxes.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Component from './entry-boxes-component';
+
+import * as actions from './entry-boxes-actions';
+
+const EntryBoxesContainer = props => {
+  const closeEntryBoxes = () => props.changeUI({ isCategoriesBoxesVisible: false });
+  const openSidebar = () => props.changeUI({ isPaddingActive: true });
```

Comment:
What do you think of changing `view` padding when this ui state is active.
Here is a [reference on ArcGis docs](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-SceneView.html#padding).
Eventually the padding would be deactivated when we enter the `fullscreen`/`clean` mode.